### PR TITLE
fix(HelpText): css alignments

### PR DESCRIPTION
**Pull request by eirikbacker**
Part of https://github.com/digdir/designsystemet/issues/2295

### CODE REVIEW

File: packages/react/src/components/HelpText/HelpText.stories.tsx [hunk 1]
```diff
@@ -13,24 +13,15 @@ const decorators = [
 ];
 
 export default {
-  title: 'Komponenter/HelpText',
+  'aria-label': 'Komponenter/HelpText',
```
Comment (Barsnes) [inline packages/react/src/components/HelpText/HelpText.stories.tsx:16]: This title needs to be here to place the component correctly in storybook
image
```suggestion
  title: 'Komponenter/HelpText',
  'aria-label': 'Komponenter/HelpText',
```

Comment (eirikbacker) [reply packages/react/src/components/HelpText/HelpText.stories.tsx]: Oh 🙈

**Review by Barsnes — CHANGES_REQUESTED** [review]
small and large icons don't render correctly:
image

**Comment by eirikbacker** [comment]
> small and large icons don't render correctly: image

shit,  bra sett - det er fikset nuh 💪

**Review by Barsnes — CHANGES_REQUESTED** [review]
Something is still off with the icon. It looks like it changes size and position when I open the helptext. This might just look like it since both colors are changing 🤔 

EDIT: I tested by holding my cursors over the top of the icon between states, and it does change.

This is medium:
image
image

small:
image
image

The hover state should also look the same as when it is active, as per the Figma design:
image

**Comment by Febakke** [comment]
I like the new behaviour where there is a dedicated hover state before the active state. 👏 I will replicate this in Figma
Sizes look great 👌 I can see that the components in Figma is not connected to our tokens, I will fix this. 

As @Barsnes pointed out I also see some issues with the "?" icon between default and active state. To me it looks like the container gets larger in som way and that changes what the `70%` size of the icon means. But im not sure. ideally the "?" should have the same size in all states.

**Comment by eirikbacker** [comment]
@Febakke @Barsnes the icon size change was an attempt to fix a half-pixel rendering issue, but found another workaround ensuring the icons is always consistent in size now ☺️ also adjusted the size to `65%` as we talked about over Figma-huddle @Febakke , so this should be ready for review again now 🙌 
Thanks for feedback and pingpong 🫶

**Comment by Febakke** [comment]
LGTM 🙌
Everything is fixed in Figma as well 🤝

File: .changeset/tame-rats-mix.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet-css": patch
+"@digdir/designsystemet-react": patch
+---
+
+HelpText: Render icon with pseudo element and require aria-label
```
Comment (Barsnes) [inline .changeset/tame-rats-mix.md:6]:
```suggestion
HelpText: 
- Use Popover API
- Remove `portal` prop
- Render icon with pseudo element and require aria-label
```

**Review by Barsnes — APPROVED** [review]

**Comment by Barsnes** [comment]
Out of the scope of this PR, but should `HelpText` also expose the `open` prop 🤔

**Comment by Barsnes** [comment]
> > Out of the scope of this PR, but should `HelpText` also expose the `open` prop 🤔
> 
> Was thinking about it too, but I would maybe dare to suggest "no" when we launch, as it is easier to add than to remove, and HelpText is in the first place a component we want people to _not_ use unless really needing to hide content, so using it _but_ having it default open would be kind of antipattern? Though, I see there can be scenarios where people would like the ability to have `open`. It's at least very easy to add now ☺️

I agree. I was thinking the `open` prop could be used to make it controlled, which I could see why some people would want. 
If it is a good pattern is a different discussion... 😅

**Pull request merged**